ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/admin/UserManagementClient.scala

```diff
@@ -52,10 +52,14 @@ final class UserManagementClient(service: UserManagementServiceStub)(implicit
       .deleteUser(proto.DeleteUserRequest(userId.toString))
       .map(_ => ())
 
-  def listUsers(token: Option[String] = None): Future[Vector[User]] =
+  def listUsers(
+      token: Option[String] = None,
+      pageToken: String,
+      pageSize: Int,
+  ): Future[Vector[User]] =
```

Comment:
We should return `nextPageToken` from `listsUser`, and ignore it (for now) in the callers

Reply:
```
ledger-service/http-json/src/main/scala/com/digitalasset/http/Endpoints.scala
daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/ledgerinteraction/GrpcLedgerClient.scala
```

Reply:
@nickchapman-da Ok, returning a §`Future[(Vector[User], String)]`